### ISSUE TITLE
Nan avg-loss get large during training

### ISSUE THREAD

**Issue by only-zzx** [AlexeyAB/darknet#7880]
have an issue with  Nan avg-loss:
while training my own dateset in p5,i found avg-loss getting bigger and bingger.
but same dataset has good result in YOLOv4 traning.
I chang the cfg parameter "batch=10" to fix my GPU
cfg-file is here:[yolov4-p5.txt](https://github.com/AlexeyAB/darknet/files/6789197/yolov4-p5.txt)

i use command:./darknet detector train cfg/voc.data cfg/yolov4-p5.cfg ./yolov4-p5.conv.232 -gpus 0,1,2,3 -map
my 
![企业微信截图_16258087963150](https://user-images.githubusercontent.com/65695179/125031431-3eaf8e00-e0bf-11eb-9a8a-bb06159df03e.png)

**Comment by lsd1994** [comment]
reduce learning rate

**Comment by only-zzx** [comment]
> reduce learning rate

my learning_rate=0.0001. have to reduce more?

**Comment by lsd1994** [comment]
I don't know you dataset and this model in detail, you can just try it.
I got the similar chart with you and just solved by reducing learning rate.

**Comment by bsysman** [comment]
Has your problem been solved，I have the same problem as you。
